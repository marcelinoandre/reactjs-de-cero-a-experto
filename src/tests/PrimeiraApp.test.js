import React from 'react';
import { render } from '@testing-library/react';
import PrimeraApp from '../PrimeraApp';
import { shallow } from 'enzyme';

describe('PrimeiraApp', () => {
  // it('should show the message  "Hello World!" ', () => {
  //   const message = 'Hello World!';
  //   const { getByText } = render(<PrimeraApp saludo={message} />);
  //   expect(getByText(message)).toBeInTheDocument();
  // });

  it('should show the message "ola mundo', () => {
    const wrapper = shallow(<PrimeraApp saludo={'ola mundo'} />);

    expect(wrapper).toMatchSnapshot();
  });
});
