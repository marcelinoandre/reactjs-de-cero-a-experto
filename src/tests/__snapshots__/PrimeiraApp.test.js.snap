// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrimeiraApp should show the message "ola mundo 1`] = `
<Fragment>
  <h1>
     
    ola mundo
     
  </h1>
  <p>
     
    Soy un subtitulo
     
  </p>
</Fragment>
`;
